presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -89,14 +107,14 @@ public boolean schemaExists(ConnectorSession session, String schema)
     @Override
     public Set<String> getSchemaNames(ConnectorSession session)
     {
-        JdbcIdentity key = JdbcIdentity.from(session);
+        SchemaNamesCacheKey key = new SchemaNamesCacheKey(JdbcIdentity.from(session), getSessionProperties(session));
         return get(schemaNamesCache, key, () -> delegate.getSchemaNames(session));
     }
 
     @Override
     public List<SchemaTableName> getTableNames(ConnectorSession session, Optional<String> schema)
     {
-        TableNamesCacheKey key = new TableNamesCacheKey(JdbcIdentity.from(session), schema);
+        TableNamesCacheKey key = new TableNamesCacheKey(JdbcIdentity.from(session), getSessionProperties(session), schema);
```

Comment:
This seems redundant. I think we can skip session props here for now.

Reply:
This was raised by @kokosing so I've added this for consistency

Reply:
@kokosing this seems redundant